src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+//applicationcdnurl=https://raw.github.com/GoogleChrome/accessibility-developer-tools/stable/dist/js/axs_testing.js
+//jquerycdnurl=http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js
+
+applicationcdnurl=http://localhost:8000/accessibility-developer-tools-master/dist/js/axs_testing.js
+jquerycdnurl=http://localhost:8000/js/jquery/1.10.2/jquery.min.js
```

Comment:
Looks like you have these local ones still sitting in here. Can we remove them and uncomment the actual ones?